TrafficCapture/trafficReplayer/src/test/resources/jolt/operations/decreaseShards.jolt

```diff
@@ -0,0 +1,20 @@
+{
+    "operation": "shift",
```

Comment:
Question - where/how is this configuration used? I only see a reference to it in `JsonTransformerTest` as a `public static` member that isn't referenced anywhere else